public/index.html

```diff
@@ -26,9 +26,8 @@
     -->
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@600;400&display=swap" rel="stylesheet">
```

Comment:
Do this: `<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap" rel="stylesheet" type='text/css'>`

For some reason, it doesn't like when the font weights aren't listed in increasing order. This will fix it. The type attribute will allow is to use CSS shorthand like `bold` which I think we already started to do.